src-tauri/tests/tests/fixtures-get_state/actions/create_new1.json

```diff
@@ -0,0 +1,3 @@
+{
+    "type": "[App] Get state"
+}
```

Comment:
For gits version control it is preferred to have a newline at the end of text files. So you can add a newline to each of your json files!

More info in case you're interested: https://medium.com/@alexey.inkin/how-to-force-newline-at-end-of-files-and-why-you-should-do-it-fdf76d1d090e